_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,151 @@
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding SC 1.2.1:Audio-only and Video-only (Prerecorded)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html)
+- [G159:Providing an alternative for time-based media for video-only content](https://www.w3.org/TR/WCAG20-TECHS/G159.html) 
+
+
+## Test Cases
+
+## Passed
+
+#### Pass example 1
+
+A silent video element with a text transcript on the same page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<p>The above video shows a giant fat rabbit climbing out of a hole in the ground. 
+He stretches, yaws, and then starts walking. 
+Then he stops to scratch his bottom.</p>
+```
+
+#### Pass example 2
+
+A silent video element with a link to a text transcript on a different page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<a href="/test-assets/rabbit-video-transcript.html">Transcript</p>
+```
+
+## Failed
+
+#### Fail example 1
+
+A silent video element with an incorrect text transcript on the same page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<p>The above video shows a giant fat dog climbing out of a hole in the ground. 
+He stretches, yaws, and then starts walking. 
+Then he stops to scratch his bottom.</p>
+```
+
+#### Fail example 2
+
+A silent video element with a link to an incorrect text transcript on a different page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<a href="/test-assets/rabbit-video-incorrect-transcript.html">Transcript</p>
+```
+
+#### Failed example 3
+
+A silent video element with an invisible text transcript on the same page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<p style="text-indent: -9999px;">The above video shows a giant fat rabbit climbing out of a hole in the ground. 
+He stretches, yaws, and then starts walking. 
+Then he stops to scratch his bottom.</p>
+```
+
+#### Failed example 4
+
```

Comment:
Missing the description for this test case

Reply:
Thanks! Added it.